synapse/storage/persist_events.py

```diff
@@ -35,6 +35,7 @@
 )
 
 import attr
+from mypy_extensions import DefaultNamedArg
```

Comment:
Is there a better way to tackle this?

The build is failing because I used it `builtins.ModuleNotFoundError: No module named 'mypy_extensions'`, https://github.com/matrix-org/synapse/runs/4261552290?check_suite_focus=true#step:6:739